src/main/webapp/common_util.js

```diff
@@ -12,17 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*
```

Comment:
Nit: resolve EsLint things, where viable and reasonable

Reply:
Did so - a few are saying unused functions, but they're just used in other files